index.html

```diff
@@ -157,62 +155,47 @@
                             <div class="avatar">
-                                {% if student.image %}
-                                <img src="images/students/{{student.image}}"
-                                alt="{{student.name}}"
+                                
+                                <img src="images/students/gautam.jpg"
+                                alt="gautam.jpg"
                                 width="200px" height="200px">
-                                {% else %}
-                                <img src="images/mentors/member.png"
-                                alt="{{student.name}}"
-                                width="200px" height="200px">
-                                {% endif %}
                                 <div class="social">
-                                    {% if student.github %}
-                                    <a href="//github.com/{{student.github}}"
+                                    <a href="http://www.github.com/gautamjajoo"
                                        target="_blank">
+                                       <img src="images/icons/github.png"
+                                alt="gautam.jpg" style="width:55px; height:50px; padding-left:-4; padding-bottom:5;">
                                         <i class="fa fa-github"></i>
                                     </a>
-                                    {% endif %}
-                                    {% if student.facebook %}
-                                    <a href="//facebook.com/{{student.facebook}}"
+                                    <a href="http://www.facebook.com/gautamjajoo"
                                        target="_blank">
+                                       <img src="images/icons/facebook.png"
+                                alt="gautam.jpg" style="width:55px; height:50px; padding-left:5; padding-bottom:5;">
                                         <i class="fa fa-facebook"></i>
                                     </a>
-                                    {% endif %}
                                     <br>
-                                    {% if student.twitter %}
-                                    <a href="//twitter.com/{{student.twitter}}"
+                                    <a href="http://www.twitter.com/gautamjajoo"
                                        target="_blank">
+                                       <img src="images/icons/twitter.png"
+                                alt="gautam.jpg" style="width:50px; height:50px; padding-bottom:5;">
                                         <i class="fa fa-twitter"></i>
                                     </a>
-                                    {% endif %}
-                                    {% if student.blog %}
-                                    <a href="//{{student.blog}}"
-                                       target="_blank">
-                                        <i class="fa fa-code"></i>
```

Comment:
You are adding the files to index.html, you should not do that, you only need to include your info to the students file and add your photo to the /images/students section, remember an image has to be 240x240 pixels [w x h]